README.md

```diff
@@ -144,6 +159,12 @@ Change the `num_inference_steps` to `30`. With this, you can get an image genera
 
 You can also experiment with reducing the `image_size`.
 
+## How to run with Google Colab?
+
+- [TensorRT →](https://colab.research.google.com/drive/1WQ98YBHTG355vL5wKbmNj9xeBmHRZGJb?usp=sharing)
+
+In each folder, we provide a notebook which run the full flow from model convertion to inference on T4 GPU on Google Colab
```

Comment:
Change to:
In each folder, we will provide a Google Colab notebook with which you can test the full flow and inference on a T4 GPU